Adding context to depletion cycle settings (#2556)
terrapower/armi: doc/user/inputs.rst

    * ``availabilityFactor(s)`` (default = 1.0)
    * ``cycleLength(s)`` (default = 365.2425)

In addition, one may optionally use the ``powerFractions`` setting to change the reactor power between each cycle. With these settings, a user can define a history in which each cycle may vary in power, length, and uptime.
The "uptime" being the amount of time the reactor is at ``power * powerFractions[i]`` for cycle ``i``.

> The uptime duration is determined by ``cycleLength[i] * availabilityFactor[i]`` days with "downtime" ``cycleLength[i] * (1 - availabilityFactor[i])``.

Reviewer: ```suggestion
The uptime duration is determined by ``cycleLengths[i] * availabilityFactors[i]`` days with "downtime" ``cycleLengths[i] * (1 - availabilityFactors[i])``.
```

> The "uptime" being the amount of time the reactor is at ``power * powerFractions[i]`` for cycle ``i``.
> The uptime duration is determined by ``cycleLengths[i] * availabilityFactors[i]`` days with "downtime" ``cycleLengths[i] * (1 - availabilityFactors[i])``.
> The cycle length is then the amount of total time (uptime + downtime) between the start of two successive cycles.

Reviewer 2: ```suggestion
The "uptime" being the amount of time the reactor is at ``power * powerFractions[i]`` for cycle ``i``. The uptime duration is determined by ``cycleLengths[i] * availabilityFactors[i]`` days with "downtime" ``cycleLengths[i] * (1 - availabilityFactors[i])``. The cycle length is then the amount of total time (uptime + downtime) between the start of two successive cycles.
```

Reviewer 2: I think these three sentences are supposed to be one paragraph, so I fixed it.  They would be rendered that way.

If they are supposed to be a list, we can make them a list. That's fine too.

Author: The lack of new line between the sentences means they are rendered as the same paragraph. From [the built PDF docs](https://github.com/terrapower/armi/actions/runs/27988077643/job/82834162022?pr=2556)

<img width="564" height="336" alt="Image" src="https://github.com/user-attachments/assets/0e7bcbb2-ccb5-4768-8190-e26332fd6682" />

> The history is restricted, however, to each cycle having a constant power, to each cycle having the same number of burnup nodes, and to those burnup nodes being evenly spaced within each cycle. An example simple cycle history might look like

Reviewer 2: I know you didn't write this, but "constant power"... I thought some people downstream were interpreting this "power" as the power of the reactor at an equilibrium state.  Am I wrong?

Author: I cannot speak for how people are interpreting the power setting. The `core.p.power` parameter is [described as](https://terrapower.github.io/armi/user/params_report.html#core-parameters)
>	Thermal power of the reactor core. Corresponds to the nuclear power generated in the core.

The above scheme would represent 3 cycles of operation:

    1. 100% power for 90 days, split into two segments of 45 days each, followed by 10 days shutdown (i.e. 90% capacity)
    2. 50% power for 30 days, split into two segments of 15 days each, followed by 70 days shutdown (i.e. 15% capacity)
    3. 100% power for 93 days, split into two segments of 46.5 days each, followed by 7 days shutdown (i.e. 93% capacity)

In each cycle, criticality calculations will be performed at 3 (``burnSteps + 1``) nodes evenly-spaced through the uptime portion of the cycle, without option for changing node spacing or frequency.

> A more detailed cycle history, may be specified through the use of the ``cycles`` setting.

Reviewer: ```suggestion
A more detailed cycle history may be specified through the use of the ``cycles`` setting.
```

> For each cycle, an entry to the ``cycles`` list is made with the following optional fields:

Reviewer 2: ```suggestion
A more detailed cycle history may be specified using the ``cycles`` setting. For each cycle, an entry to the ``cycles`` list is made with the following optional fields:
```

Reviewer 2: Just tightening up the language a little. I won't die on this hill.
